File: API/Features/Users/Dtos/UserReadDto.cs
using API.Infrastructure.Classes;

namespace API.Features.Users {

    public class UserReadDto {

        public string Id { get; set; }
        public string UserName { get; set; }
        public string Displayname { get; set; }
        public SimpleEntity Customer { get; set; }
        public string Email { get; set; }
        public bool IsAdmin { get; set; }
        public bool IsActive { get; set; }

    }

}

File: FrontEnd/src/app/features/users/classes/dtos/user-read-dto.ts
import { Guid } from 'guid-typescript'
// Custom
import { CustomerAutoCompleteVM } from '../../../customers/classes/view-models/customer-autocomplete-vm'

export class UserReadDto {

    id: Guid
    username: string
    displayname: string
    customer: CustomerAutoCompleteVM
    email: string
    isFirstFieldFocused: boolean
    isAdmin: boolean
    isActive: boolean

}
